src/map/status.c

```diff
@@ -3176,6 +3176,8 @@ bool status_calc_weight(struct map_session_data *sd, enum e_status_calc_weight_o
 			sd->max_weight += sd->max_weight * sc->data[SC_KNOWLEDGE]->val1 / 10;
 		if ((skill = pc_checkskill(sd, ALL_INCCARRY)) > 0)
 			sd->max_weight += 2000 * skill;
+		if (pc_ismadogear(sd))
+			sd->max_weight += 1500;
```

Comment:
@aleos89 i tried to apply this, seems only increase 150 weight ingame
maybe this? `sd->max_weight += 15000;`